weave/op_policy.py

```diff
@@ -40,6 +40,8 @@
     "gqlroot-wbgqlquery",
 ] + CACHE_AND_PARALLEL_OP_NAMES
 
+ARROW_FS_OPS = ["run-history3", "run-history3_with_columns"]
```

Comment:
I wonder if we would also want to track table-rows?

Reply:
@shawnlewis said let's start with history3 for now, and then we can add